documentation/faq.mdx

```diff
@@ -105,17 +133,23 @@ Your GPU might be too weak. Some of our rendering code is a bit inefficient and
 
 ### The game crashed!!
 
-Curses! Contact us in the help channel on the discord server and tell us about it! Post the log files as well. These are located in the `data/log` folder of the release (if you are using the Launcher, there is a button on the settings tab that takes you to the folder containing the `data` folder).
-
-Make sure you tell us exactly how the crash happened and what settings you were using.
+Curses! Contact us in the [help channel](https://discord.com/channels/756287461377703987/1019706639810580553) on the discord server and tell us about it! Post a support package as well. If you're using a Launcher, it can be generated under the help section. Make sure you tell us exactly how the crash happened and what settings you were using.
+<!--https://discord.com/channels/756287461377703987/1020343516242653214-->
 
 ### My controller doesn't work!
 
 Our controller handling does need some work, but generally speaking any controller defined in this database should be supported https://github.com/gabomdq/SDL_GameControllerDB You can try the following until support improves:
 
-- Make sure you only have 1 controller plugged in at a time to ensure that one is used
-- Try leveraging Steam to get the controller recognized
-- If your controller isn't in the DB above, you should follow the steps there to make a mapping for your controller
+- Make sure you only have 1 controller plugged in at a time to ensure that one is used.
+- Try leveraging Steam to get the controller recognized. Conversely, if you have steam or any other games that may be using your controller open, try closing them.
```

Comment:
capitalize Steam.
It might be a good idea to specifically call out the desktop layout for Steam Input here?

Reply:
I'm sorry, but I don't have Steam, so I may end up "talking about something without knowing what I'm talking about" again :p I found [this forum post](https://steamcommunity.com/app/1675200/discussions/1/3498761139323172659/), but honestly, I'm not following it. (Steam Deck's built-in controls? change in controller layout == detected controller plugged in? The Desktop controller layout == Keyboard and mouse perhaps?) Can you elaborate specifically on what you'd like to see there?

Reply:
Steam Input is a feature that Steam has to "improve" controller compatibility, including letting you use your controller to control the entire desktop (Desktop Layout), which does things like mapping the X button on a PlayStation controller to Enter on your keyboard, etc.

Reply:
![image](https://github.com/open-goal/open-goal.github.io/assets/2515356/6439a4c1-91e9-42ce-9d61-4077c069d4bc)

In the popup from clicking this button is where you can disable "Steam Input" which tries to make your controller function as a replacement for mouse+keyboard (which introduces more button mappings)